### PR TITLE
Fix ACR repository name

### PR DESCRIPTION
**Pull request by Stretch96**
* The image name (`image_name`) has been set to `aca-app` rather than `academies-academisation-api` within the Azure Container Apps terraform module input variables.

### CODE REVIEW

File: .github/workflows/build-and-push-image-development.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Prepare tags
         id: prepare-tags
         run: |
-          DOCKER_IMAGE=${{ secrets.DEVELOPMENT_ACR_URL }}/academies-academisation-api
```
Comment (jordan-beard) [inline .github/workflows/build-and-push-image-development.yml:26]: That name doesn't seem anywhere near as descriptive.
Why the change?

Comment (Stretch96) [reply .github/workflows/build-and-push-image-development.yml:26]: It's how I initially launched the infrastructure unfortunately :( ...
Changing the name to `academies-academisation-api` now, will cause resources to be recreated and we'll have to reconfigure the apps that are connecting to the api
`aca` was initially chosen, as that was how the service was being refered to

**Review by jordan-beard — APPROVED** [review]
